lib/Cro/WebApp/Template.pm6

```diff
@@ -57,13 +57,14 @@ multi render-template(Str $template, $initial-topic, :%parts --> Str) is export
 
     # Finally, render it.
     Cro::WebApp::LogTimeline::RenderTemplate.log: :$template, {
-        render-internal($compiled-template, $initial-topic, %parts)
+        render-internal($template, $compiled-template, $initial-topic, %parts)
     }
 }
 
-sub render-internal($compiled-template, $initial-topic, %parts) {
+sub render-internal($template, $compiled-template, $initial-topic, %parts) {
     my $*CRO-TEMPLATE-MAIN-PART := $initial-topic;
     my %*CRO-TEMPLATE-EXPLICIT-PARTS := %parts;
+    my $*TEMPLATE-FILE = $template;
```

Comment:
I'm sure this is going to give wrong results when we call a sub from a `use`'d template.